GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+
+# Project Governance
+
+<!-- TOC -->
+
+* [Collaborators](#collaborators)
+  * [Collaborator activities](#collaborator-activities)
+  * [Collaborator nominations](#collaborator-nominations)
+* [Technical steering committee](#technical-steering-committee)
+  * [TSC meetings](#tsc-meetings)
+  * [Consensus seeking process](#consensus-seeking-process)
+* [Release Manager](#release-manager)
+
+<!-- /TOC -->
+
+
+## Collaborators
+
+Core collaborators for Systelab libraries are responsible for the GitHub repository maintenance. 
+All members of @systelab/jee-team are potential collaborators to this library.
+
+Collaborators have:
+
+* Commit access to the repository in any branch different from master.
+* Access to the Continuous integration (CI) jobs
+
+Both collaborators and non-collaborators may propose changes to the library source code. 
+The mechanism to propose such an implementation change is a GitHub pull request.
+
+Collaborators review and merge (_land_) pull requests.
+
+Three collaborators must approve a pull request before the pull request can land. 
```

Comment:
Maybe 3 is too much, at least on early stages of a library. I would suggest moving this to just 2 as it is done in other libraries.